all/entrypoint.sh

```diff
@@ -172,6 +174,10 @@ while test $# -gt 0; do
             CSHARP_OPT=$1
             shift
             ;;
+        ----with-swagger-json-names)
```

Comment:
Think there are actually too many `-` here, should probably be
```suggestion
        --with-swagger-json-names)
```

Reply:
thanks!

Reply:
@aldelucca1  should be all set. passes build check